packages/switch/src/react/index.tsx

```diff
@@ -26,6 +28,7 @@ interface SwitchProps
   name?: string
   onBlur?: React.FocusEventHandler
   onClick?: (checked: boolean) => void
+  onChange?: DefaultChangeFn
```

Comment:
Instead of adding another prop to the interface, I would prefer we just kept it on the component as you have it, but just remove it from the props/interface. 

That way, the component API doesn't change and you don't have to worry about updating the docs as well.

Reply:
Sounds good - just pushed!